app/views/DR/HomeScreen/index.js

```diff
@@ -111,6 +129,10 @@ class HomeScreen extends Component {
   };
 
   async componentDidMount() {
+    setTimeout(() => {
+      this.setState({ statusVisible: false });
+    }, 3000);
```

Comment:
```suggestion
    setTimeout(() => {
      this.setState({ statusVisible: false });
    }, 7000);
```
I think that to be a visual notification they pass very quickly, perhaps it is recommended to increase it to 5-7 seconds, to consider it as a notification